src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -141,7 +152,10 @@ class Simplex_tree {
   };
   struct Filtration_simplex_base_dummy {
     Filtration_simplex_base_dummy() {}
-    void assign_filtration(Filtration_value GUDHI_CHECK_code(f)) { GUDHI_CHECK(f == 0, "filtration value specified for a complex that does not store them"); }
+    void assign_filtration(Filtration_value GUDHI_CHECK_code(f))
+    {
+      GUDHI_CHECK(f == 0, "filtration value specified for a complex that does not store them");
+    }
```

Comment:
We should probably try to do some big automatic reformatting of files at a time where noone is working on them. But that's precisely the moment when nobody cares about reformatting them... :man_shrugging:

Reply:
Yes :/
We should perhaps just decide on a day and give out a warning. Because some files clearly don't follow the Gudhi guide lines anymore. It could also be the occasion to fine tune the clang-format file.